<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tab_bar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>601</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;Times New Roman&quot;;
background-color: qlineargradient(spread:pad, x1:0.224031, y1:0.239, x2:1, y2:1, stop:0.0965909 rgba(227, 217, 109, 255), stop:1 rgba(255, 255, 255, 255));</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="elideMode">
    <enum>Qt::ElideLeft</enum>
   </property>
   <property name="documentMode">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="home_tab">
    <attribute name="title">
     <string>Home</string>
    </attribute>
    <widget class="QLabel" name="welcome_label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>201</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 30pt &quot;Times New Roman&quot;;
color: rgb(97, 97, 97);
background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>Welcome,</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="name_label">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>0</y>
       <width>401</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 30pt &quot;Times New Roman&quot;;
background-color: rgba(255, 255, 255, 0);
color: rgb(97, 97, 97);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="total_label">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>450</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 20pt &quot;Times New Roman&quot;;
text-decoration: underline;
color: rgb(97, 97, 97);
background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>Total:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="total_money_label">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>490</y>
       <width>311</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 20pt &quot;Times New Roman&quot;;
color: rgb(97, 97, 97);
background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>70</y>
       <width>641</width>
       <height>371</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="title">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="new_month_button">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>10</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(245, 245, 245);</string>
     </property>
     <property name="text">
      <string>New Month</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="income_tab">
    <attribute name="title">
     <string>Income</string>
    </attribute>
    <widget class="QLabel" name="amount_label">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>180</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 20pt &quot;Times New Roman&quot;;;
color: rgb(97, 97, 97);
background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>Amount:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="description_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 20pt &quot;Times New Roman&quot;;;
color: rgb(97, 97, 97);
background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>Description:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="income_amount_edit">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>190</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="income_description_edit">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>250</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="add_income_button">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>340</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(59, 213, 13);</string>
     </property>
     <property name="text">
      <string>Add Income</string>
     </property>
    </widget>
    <widget class="QLabel" name="income_label">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>50</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 20pt &quot;Times New Roman&quot;;;
color: rgb(68, 203, 0);
background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>Income</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QScrollArea" name="income_scroll_bar">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>90</y>
       <width>221</width>
       <height>451</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>219</width>
        <height>449</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="total_label_2">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>500</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15pt &quot;Times New Roman&quot;;;
color: rgb(97, 97, 97);
background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>Total:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="total_amount_label">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>500</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 0);
font: 75 16pt &quot;Times New Roman&quot;;
color: rgb(68, 223, 25);
border-color: rgb(70, 70, 70);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="income_error_label">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>290</y>
       <width>371</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 0);
font: 10pt &quot;Times New Roman&quot;;
color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="expenses_tab">
    <attribute name="title">
     <string>Expenses</string>
    </attribute>
    <widget class="QLabel" name="amount_label_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>180</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 20pt &quot;Times New Roman&quot;;;
color: rgb(97, 97, 97);
background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>Amount:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="description_label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 20pt &quot;Times New Roman&quot;;;
color: rgb(97, 97, 97);
background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>Description:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="expenses_amount_edit">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>190</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="expenses_description_edit">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>250</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="add_expense_button">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>340</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(211, 0, 0);</string>
     </property>
     <property name="text">
      <string>Add Expense</string>
     </property>
    </widget>
    <widget class="QLabel" name="total_label_3">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>500</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15pt &quot;Times New Roman&quot;;;
color: rgb(97, 97, 97);
background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>Total:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="expense_label">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>50</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 20pt &quot;Times New Roman&quot;;;
color: rgb(255, 0, 0);
background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>Expenses</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QScrollArea" name="expense_scroll_bar">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>90</y>
       <width>221</width>
       <height>451</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>219</width>
        <height>449</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="total_amount_label_2">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>500</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 0);
font: 75 16pt &quot;Times New Roman&quot;;
color: rgb(255, 0, 0);
border-color: rgb(70, 70, 70);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="expense_error_label">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>290</y>
       <width>371</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 0);
font: 10pt &quot;Times New Roman&quot;;
color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="account_tab">
    <attribute name="title">
     <string>Account</string>
    </attribute>
    <widget class="QLabel" name="my_account_label">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>30</y>
       <width>211</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 30pt &quot;Times New Roman&quot;;
color: rgb(97, 97, 97);
background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>My Account</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="first_name_label">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>150</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15pt &quot;Times New Roman&quot;;;
color: rgb(97, 97, 97);
background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>First Name:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="last_name_label">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>190</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15pt &quot;Times New Roman&quot;;;
color: rgb(97, 97, 97);
background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>Last Name:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="email_label">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>230</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15pt &quot;Times New Roman&quot;;;
color: rgb(97, 97, 97);
background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>Email:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="password_label">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>270</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15pt &quot;Times New Roman&quot;;;
color: rgb(97, 97, 97);
background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>Password:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="first_name_box">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>150</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15pt &quot;Times New Roman&quot;;;
color: rgb(97, 97, 97);
background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="last_name_box">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>190</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15pt &quot;Times New Roman&quot;;;
color: rgb(97, 97, 97);
background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="email_box">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>230</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15pt &quot;Times New Roman&quot;;;
color: rgb(97, 97, 97);
background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="password_label_2">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>270</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15pt &quot;Times New Roman&quot;;;
color: rgb(97, 97, 97);
background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="delete_account_button">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>400</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(211, 0, 0);</string>
     </property>
     <property name="text">
      <string>Delete Account</string>
     </property>
    </widget>
    <widget class="QPushButton" name="change_password_button">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>400</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(186, 186, 186);</string>
     </property>
     <property name="text">
      <string>Change Password</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
